clientv3/watch.go

```diff
@@ -52,6 +58,27 @@ type WatchResponse struct {
 	Canceled bool
 }
 
+// IsGoneEvent tells from the event if the key is gone, either deleted or expired.
+func IsGoneEvent(e *storagepb.Event) bool {
```

Comment:
Actually we never return EXPIRE. we will probably just remove expire type in api.